components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -3988,14 +3990,17 @@ public static String getMountedPath(RegistryContext registryContext, String path
             handleException("Invalid resource Path " + resourceUrl);
         }
         Resource apiDocResource;
-        Registry registryType;
-        int tenantId;
-        String tenantDomain = MultitenantUtils.getTenantDomain(userName);
+        Registry registryType = null;
+        int tenantId = MultitenantConstants.SUPER_TENANT_ID;
         try {
+            if (tenantDomain != null && !"null".equals(tenantDomain)) {
```

Comment:
can we get the null string as well. If not use StringUtils.isEmpty() will handle null and empty case